docs/topics/execute.adoc

```diff
@@ -68,6 +68,49 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 $ __RHAMT_HOME__/bin/rhamt-cli -Dwindup.decompiler=procyon --input __INPUT_ARCHIVE_OR_DIRECTORY__ --output __OUTPUT_REPORT_DIRECTORY__ --target __TARGET_TECHNOLOGY__ --packages __PACKAGE_1__ __PACKAGE_2__
 ----
 
+[discrete]
+[[cli_bash_completion]]
+== {ProductShortName} {CLINameTitle} Bash Completion
+
+The {ProductShortName} {CLIName} includes a set of bash completions, allowing the xref:command_line_arguments[{ProductShortName} Command-line Arguments] to be auto completed by pressing Tab when entering the commands. For instance, entering the following displays a list of available arguments.
+
+[source,options="nowrap",subs="+quotes"]
+----
+$ __RHAMT_HOME__/bin/rhamt-cli [TAB]
+----
+
+[discrete]
+[[bash_completion_temporary]]
+=== Enable Bash Completion
+
+To enable bash completion for the current shell execute the following command. After the prompt returns follow the steps in xref:execute[Run the {CLIName}] as normal.
+
+[source,options="nowrap",subs="+quotes"]
+----
+$ . __RHAMT_HOME__/bash-completion/rhamt-cli
+----
+
+[discrete]
+[[bash_completion_persistent]]
+=== Enable Persistent Bash Completion
+
+The following commands allow bash completion to be persistant across restarts; however, if bash completion is desired for the current shell then the steps in xref:bash_completion_temporary[Enable Bash Completion] must be followed.
+
+* To enable bash completion for a specific user across system restarts, include the following line in that user's `~/.bashrc` file.
++
+[source,options="nowrap",subs="+quotes"]
+----
+source __RHAMT_HOME__/bash-completion/rhamt-cli
```

Comment:
This needs the prompt in front, "$" I assume.

Reply:
This is the line that should be added to a user's .bashrc, so it's not a command being executed.  I don't think the $ is necessary in this case, then.

Reply:
Oops, my apologies, you're right :)